### DISCUSSION TITLE
supporting Deepfloyd IF

### DISCUSSION THREAD

**Discussion by omar92** [comfyanonymous/ComfyUI#592] [Ideas]
this things is massive , have high hope for it , and would love to see it supported in comfyUI

resources:
https://www.youtube.com/watch?v=4Zkipll5Rjc
https://huggingface.co/DeepFloyd/IF-I-XL-v1.0
https://github.com/deep-floyd/IF

**Comment by fractal-fumbler** [comment]
https://github.com/comfyanonymous/ComfyUI/issues/590#issuecomment-1529109662

**Reply by fractal-fumbler** [reply]
https://github.com/Zuellni/ComfyUI-Custom-Nodes#deepfloyd-nodes

**Reply by rethink-studios** [reply]
How would you install these checkpoints for Comfy?

**Reply by Zuellni** [reply]
My 'implementation' uses diffusers and the checkpoints are downloaded from the huggingface repo. I believe you need to accept their terms of use on the [hf page](https://huggingface.co/DeepFloyd) to get access, and that also requires an account. Not ideal but it is what it is.
